template/src/Autovis/Controls.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
```

Comment:
please use the same quote style that i'm using elsewhere in this repo. (single quotes in js, double quotes only in react component string properties like className="asdf")